.coderabbit.yml

```
@@ -11,7 +11,7 @@ reviews:
   auto_review:
     enabled: true
     drafts: false
-  path_filters:
-    - !package.json
+  path_filters: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)